src/components/AppProvider/tests/AppProvider.test.tsx

```diff
@@ -1,9 +1,20 @@
 import React, {useContext} from 'react';
+import {matchMedia} from '@shopify/jest-dom-mocks';
 import {mountWithAppProvider} from 'test-utilities/legacy';
+import {mountWithApp} from 'test-utilities/react-testing';
+import {MediaQueryProvider} from 'components/MediaQueryProvider';
 import {LinkContext} from '../../../utilities/link';
 import {AppProvider} from '../AppProvider';
 
 describe('<AppProvider />', () => {
+  beforeEach(() => {
```

Comment:
AppProvider renders `MediaQueryProvider` now which uses matchMedia and since jsdom doesn't provide it, we'll need it mock it.